index.html

```diff
@@ -618,6 +618,94 @@ <h4>
         </section>
       </section>
     </section>
+    <section>
+      <h2>
+        Automation
+      </h2>
+      <p>
+        The Device Posture API pose a challenge to test authors, as fully
+        exercising interface requires physical hardware devices. To address this
+        challenge this document defines a [[WEBDRIVER2]] [=extension commands=]
+        that allows users to control the reported device posture and simulate a
+        real device.
+      </p>
+      <h3>
+        Extension Commands
+      </h3>
+      <h4>
+        Set device posture
+      </h4>
+      <table class="simple">
+        <tr>
+          <th>
+            HTTP Method
+          </th>
+          <th>
+            [=extension command URI Template|URI Template=]
+          </th>
+        </tr>
+        <tr>
+          <td>
+            POST
+          </td>
+          <td>
+            /session/{session id}/deviceposture
+          </td>
+        </tr>
+      </table>
+      <p>
+        This [=extension commands|extension command=] changes device posture to a specific
+        {{DevicePostureType}}.
+      </p>
+      <table class="simple">
+        <caption>
+          Properties of the parameters argument used by this algorithm
+        </caption>
+        <tr>
+          <th>
+            Parameter name
+          </th>
+          <th>
+            Value type
+          </th>
+          <th>
+          Required
+          </th>
+        </tr>
+        <tr>
+          <td>
+            posture
+          </td>
+          <td>
+            String
+          </td>
+          <td>
+            yes
+          </td>
+        </tr>
+      </table>
+      <p>
+        The [=remote end steps=] are:
+      </p>
+      <ol class="algorithm" data-cite="WEBDRIVER2">
+        <li>Let |posture| be the result of invoking get a property "posture"
+        from |parameters|.
+        </li>
+        <li>If |posture| is neither {{DevicePostureType/continuous}} nor
+        {{DevicePostureType/folded}}, return [=error=] with
+        [=error code|WebDriver error code=] [=invalid argument=].
+        </li>
```

Comment:
You asked somewhere if I meant `{{foo/bar}}` or `"{{foo/bar}}"` here, and it's the latter. You are referencing strings, so you should follow https://infra.spec.whatwg.org/#strings (note that a lot of specs do not follow the monospace font recommendation, the double quotes part is more important). See all the strings referenced in https://w3c.github.io/webdriver/#print-page for example.

Reply:
Fixed in https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/256310eb663523fe4e7ba3c10c777f429310c395